ChangeLog

```diff
@@ -38,8 +38,6 @@
   (contributed by @vimpostor)
 
 - The term "Central server" has been replaced with "Directory server" (#1407).
-  Note that the program will still accept the --centralserver option for backward
-  compatibility with existing system startup scripts.
```

Comment:
Hrm, might be an indicator for a partially bad merge/rebase?

Reply:
My bas. Will solve.

Reply:
Solved by 
5aa0168